packages/clerk-js/src/core/resources/OrganizationMembership.ts

```diff
@@ -40,7 +40,7 @@ export class OrganizationMembership extends BaseResource implements Organization
     }).then(res => {
       // TODO: Fix typing
       const { data: suggestions, total_count } =
-        res?.response as unknown as ClerkPaginatedResponse<OrganizationMembershipJSON>;
+        res?.response as unknown as ClerkPaginatedResponse<OrganizationMembershipJSON>; // oxlint-disable-line no-unsafe-optional-chaining
```

Comment:
should we maybe make this rule a warning (perhaps specifically for `clerk-js`) instead of silencing it?